chore(deps): avoid 'path' issues with @rnx-kit tools

tido64 opened the pull request:
## Please verify the following:

- [x] `yarn ci:test` **jest** tests pass with new tests, if relevant
- [x] `README.md` has been updated with your changes, if relevant

## Describe your PR

This is a follow-up to the [live stream](https://www.youtube.com/watch?v=-6oCo7lTRTw) with Adam from a few weeks back. I actually found the issue the same night, but forgot to submit it over the weekend.

Some of the tools are relying on a new behaviour introduced in `@rnx-kit/tools-node` 1.2.7. Things broke because Yarn decided that the older version should be hoisted instead.

tido64 commented:
cc @afoxman @jamonholmgren

jamonholmgren commented:
Beautiful, thanks @tido64.

Do tests pass locally? They seem to be timing out in CI.

tido64 commented:
> Beautiful, thanks @tido64.
> 
> Do tests pass locally? They seem to be timing out in CI.

This is what I see locally:

<details>
<summary>Test failure log</summary>

```
 FAIL  test/vanilla/ignite-new.test.ts (92.699 s)
  ● Igniting new app! 🔥
Go get a coffee or something. This is gonna take a while. › ignite new Foo

    expect(received).toContain(expected) // indexOf

    Expected substring: "Ignite CLI ignited Foo"
    Received string:    " ignite command: /usr/local/bin/node /~/bin/ignite new Foo --debug
     [object Object]·····
       · · · · · · · · · · · · · · · · · · 🔥 Ignite 🔥 · · · · · · · · · · · · · · · · · ··
        █ Creating Foo using Ignite 7.10.9
        █ Powered by Infinite Red - https://infinite.red
        █ Using ignite-cli with pnpm
        █ Bundle identifier: com.Foo
        ────────────────────────────────────────────────·
     /private/var/folders/qc/zygl93h53zb3ctbyw0f0jbzm0000gn/T/ignite-27d4ee3ff9a300fb15dec7abdb4a939b
     /private/var/folders/qc/zygl93h53zb3ctbyw0f0jbzm0000gn/T/ignite-27d4ee3ff9a300fb15dec7abdb4a939b/Foo/.gitignore
     Progress: resolved 1, reused 0, downloaded 0, added 0···
       ╭─────────────────────────────────────────────────────────────────╮
       │                                                                 │
       │                Update available! 6.32.3 → 7.1.5.                │
       │   Changelog: https://github.com/pnpm/pnpm/releases/tag/v7.1.5   │
       │                 Run pnpm add -g pnpm to update.                 │
       │                                                                 │
       │     Follow @pnpmjs for updates: https://twitter.com/pnpmjs      │
       │                                                                 │
       ╰─────────────────────────────────────────────────────────────────╯··
     Progress: resolved 33, reused 4, downloaded 17, added 0·
     Progress: resolved 54, reused 5, downloaded 34, added 0·
     Progress: resolved 67, reused 6, downloaded 52, added 0·
     Progress: resolved 145, reused 37, downloaded 92, added 0·
     Progress: resolved 221, reused 86, downloaded 119, added 0·
      WARN  deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.·
     Progress: resolved 302, reused 142, downloaded 137, added 0·
      WARN  deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.·
     Progress: resolved 374, reused 189, downloaded 172, added 0·
      WARN  deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.·
     Progress: resolved 456, reused 252, downloaded 188, added 0·
      WARN  deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.·
     Progress: resolved 612, reused 372, downloaded 222, added 0·
     Progress: resolved 732, reused 411, downloaded 283, added 0·
      WARN  deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.·
      WARN  deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.·
     Progress: resolved 798, reused 443, downloaded 334, added 0·
     Progress: resolved 847, reused 474, downloaded 348, added 0·
      WARN  deprecated sane@4.1.0: some dependency vulnerabilities fixed, support for node < 10 dropped, and newer ECMAScript syntax/features added·
     Progress: resolved 1021, reused 593, downloaded 393, added 0·
      WARN  deprecated querystring@0.2.1: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.·
      WARN  deprecated popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1·
     Progress: resolved 1206, reused 677, downloaded 483, added 0·
      WARN  deprecated core-js@1.2.7: core-js@<3.4 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Please, upgrade your dependencies to the actual version of core-js.·
      WARN  deprecated axios@0.16.2: Critical security vulnerability fixed in v0.21.1. For more information, see https://github.com/axios/axios/pull/3410·
      WARN  deprecated set-value@1.0.0: Critical bug fixed in v3.0.1, please upgrade to the latest version.·
     Progress: resolved 1312, reused 750, downloaded 529, added 0·
      WARN  deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated·
      WARN  deprecated chokidar@2.1.8: Chokidar 2 does not receive security updates since 2019. Upgrade to chokidar 3 with 15x fewer dependencies·
      WARN  deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.·
     Progress: resolved 1436, reused 836, downloaded 553, added 0·
      WARN  deprecated chokidar@2.1.8: Chokidar 2 does not receive security updates since 2019. Upgrade to chokidar 3 with 15x fewer dependencies·
      WARN  deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.·
      WARN  deprecated highlight.js@9.18.5: Support has ended for 9.x series. Upgrade to @latest·
     Progress: resolved 1601, reused 931, downloaded 627, added 0·
      WARN  deprecated set-value@0.4.3: Critical bug fixed in v3.0.1, please upgrade to the latest version.·
      WARN  deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.·
      WARN  deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated·
      WARN  deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated·
      WARN  deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated·
     Progress: resolved 1694, reused 1003, downloaded 646, added 0·
      WARN  deprecated popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1·
      WARN  deprecated highlight.js@9.13.1: Version no longer supported. Upgrade to @latest·
      WARN  deprecated uglify-es@3.3.9: support for ECMAScript is superseded by `uglify-js` as of v3.13.0·
      WARN  deprecated set-value@1.0.0: Critical bug fixed in v3.0.1, please upgrade to the latest version.·
     Progress: resolved 1783, reused 1044, downloaded 695, added 0·
     Progress: resolved 1904, reused 1114, downloaded 751, added 0·
     Packages: +1914
    ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++·
     Packages are hard linked from the content-addressable store to the virtual store.
      Content-addressable store is at: ~/.pnpm-store/v3
      Virtual store is at:             node_modules/.pnpm·
     Progress: resolved 1904, reused 1114, downloaded 785, added 63·
     Progress: resolved 1904, reused 1114, downloaded 789, added 297·
     Progress: resolved 1904, reused 1114, downloaded 789, added 397·
     Progress: resolved 1904, reused 1114, downloaded 789, added 619·
     Progress: resolved 1904, reused 1114, downloaded 790, added 724·
     Progress: resolved 1904, reused 1114, downloaded 790, added 961·
     Progress: resolved 1904, reused 1114, downloaded 790, added 1000·
     Progress: resolved 1904, reused 1114, downloaded 790, added 1233·
     Progress: resolved 1904, reused 1114, downloaded 790, added 1443·
     Progress: resolved 1904, reused 1114, downloaded 790, added 1562·
     Progress: resolved 1904, reused 1114, downloaded 790, added 1712·
     Progress: resolved 1904, reused 1114, downloaded 790, added 1774·
     Progress: resolved 1904, reused 1114, downloaded 790, added 1836·
     Progress: resolved 1904, reused 1114, downloaded 790, added 1914, done·
     .../node_modules/dtrace-provider install$ node-gyp rebuild || node suppress-error.js·
     .../node_modules/core-js-pure postinstall$ node -e \"try{require('./postinstall')}catch(e){}\"·
     .../node_modules/ejs postinstall$ node ./postinstall.js·
     .../node_modules/core-js postinstall$ node -e \"try{require('./postinstall')}catch(e){}\"·
     .../node_modules/fsevents install$ node install.js·
     .../node_modules/core-js-pure postinstall: Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!·
     .../node_modules/core-js-pure postinstall: The project needs your help! Please consider supporting of core-js:·
     .../node_modules/core-js-pure postinstall: > https://opencollective.com/core-js··
     .../node_modules/core-js-pure postinstall: > https://patreon.com/zloirock··
     .../node_modules/core-js-pure postinstall: > bitcoin: bc1qlea7544qtsmj2rayg0lthvza9fau63ux0fstcz··
     .../node_modules/core-js-pure postinstall: Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)·
     .../node_modules/ejs postinstall: Thank you for installing EJS: built with the Jake JavaScript build tool (https://jakejs.com/)·
     .../node_modules/core-js-pure postinstall: Done·
     .../node_modules/ejs postinstall: Done·
     .../node_modules/highlight.js postinstall$ node deprecated.js·
     .../node_modules/core-js postinstall: Done·
     .../node_modules/postinstall-prepare postinstall$ node ./run.js·
     .../node_modules/highlight.js postinstall: -*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*·
     .../node_modules/highlight.js postinstall:··
     .../node_modules/highlight.js postinstall:   Verion 9 of Highlight.js has reached EOL.  It will no longer·
     .../node_modules/highlight.js postinstall:   be supported or receive security updates in the future.·
     .../node_modules/highlight.js postinstall:   Please upgrade to version 10 or encourage your indirect·
     .../node_modules/highlight.js postinstall:   dependencies to do so.·
     .../node_modules/highlight.js postinstall:   For more info:·
     .../node_modules/highlight.js postinstall:····
     .../node_modules/highlight.js postinstall:   https://github.com/highlightjs/highlight.js/issues/2877·
     .../node_modules/highlight.js postinstall:   https://github.com/highlightjs/highlight.js/blob/master/VERSION_10_UPGRADE.md·
     .../node_modules/highlight.js postinstall:···
     .../node_modules/highlight.js postinstall: -*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*·
     .../node_modules/highlight.js postinstall: Done·
     .../node_modules/postinstall-prepare postinstall: Done·
     .../node_modules/dtrace-provider install: gyp info it worked if it ends with ok·
     .../node_modules/fsevents install: gyp info it worked if it ends with ok·
     .../node_modules/fsevents install: gyp info using node-gyp@8.4.1·
     .../node_modules/fsevents install: gyp info using node@14.19.0 | darwin | x64·
     .../node_modules/dtrace-provider install: gyp info using node-gyp@8.4.1·
     .../node_modules/dtrace-provider install: gyp info using node@14.19.0 | darwin | x64·
     .../node_modules/dtrace-provider install: gyp info find Python using Python version 3.9.13 found at \"/usr/local/opt/python@3.9/bin/python3.9\"·
     .../node_modules/fsevents install: gyp info find Python using Python version 3.9.13 found at \"/usr/local/opt/python@3.9/bin/python3.9\"·
     .../node_modules/fsevents install: gyp info spawn /usr/local/opt/python@3.9/bin/python3.9·
     .../node_modules/fsevents install: gyp info spawn args [·
     .../node_modules/dtrace-provider install: gyp info spawn /usr/local/opt/python@3.9/bin/python3.9
    .../node_modules/dtrace-provider install: gyp info spawn args [
    .../node_modules/dtrace-provider install: gyp info spawn args   '/usr/local/lib/node_modules/pnpm/dist/node_modules/node-gyp/gyp/gyp_main.py',·
     .../node_modules/fsevents install: gyp info spawn args   '/usr/local/lib/node_modules/pnpm/dist/node_modules/node-gyp/gyp/gyp_main.py',·
     .../node_modules/fsevents install: gyp info spawn args   'binding.gyp',·
     .../node_modules/fsevents install: gyp info spawn args   '-f',·
     .../node_modules/fsevents install: gyp info spawn args   'make',·
     .../node_modules/fsevents install: gyp info spawn args   '-I',·
     .../node_modules/fsevents install: gyp info spawn args   '/private/var/folders/qc/zygl93h53zb3ctbyw0f0jbzm0000gn/T/ignite-27d4ee3ff9a300fb15dec7abdb4a939b/Foo/node_modules/.pnpm/registry.npmjs.org+fsevents@1.2.13/node_modules/fsevents/build/config.gypi',·
     .../node_modules/fsevents install: gyp info spawn args   '-I',·
     .../node_modules/fsevents install: gyp info spawn args   '/usr/local/lib/node_modules/pnpm/dist/node_modules/node-gyp/addon.gypi',·
     .../node_modules/fsevents install: gyp info spawn args   '-I',·
     .../node_modules/fsevents install: gyp info spawn args   '~/Library/Caches/node-gyp/14.19.0/include/node/common.gypi',·
     .../node_modules/fsevents install: gyp info spawn args   '-Dlibrary=shared_library',·
     .../node_modules/fsevents install: gyp info spawn args   '-Dvisibility=default',·
     .../node_modules/fsevents install: gyp info spawn args   '-Dnode_root_dir=~/Library/Caches/node-gyp/14.19.0',·
     .../node_modules/fsevents install: gyp info spawn args   '-Dnode_gyp_dir=/usr/local/lib/node_modules/pnpm/dist/node_modules/node-gyp',·
     .../node_modules/fsevents install: gyp info spawn args   '-Dnode_lib_file=~/Library/Caches/node-gyp/14.19.0/<(target_arch)/node.lib',·
     .../node_modules/fsevents install: gyp info spawn args   '-Dmodule_root_dir=/private/var/folders/qc/zygl93h53zb3ctbyw0f0jbzm0000gn/T/ignite-27d4ee3ff9a300fb15dec7abdb4a939b/Foo/node_modules/.pnpm/registry.npmjs.org+fsevents@1.2.13/node_modules/fsevents',·
     .../node_modules/fsevents install: gyp info spawn args   '-Dnode_engine=v8',·
     .../node_modules/fsevents install: gyp info spawn args   '--depth=.',·
     .../node_modules/fsevents install: gyp info spawn args   '--no-parallel',·
     .../node_modules/dtrace-provider install: gyp info spawn args   'binding.gyp',·
     .../node_modules/dtrace-provider install: gyp info spawn args   '-f',·
     .../node_modules/dtrace-provider install: gyp info spawn args   'make',·
     .../node_modules/dtrace-provider install: gyp info spawn args   '-I',·
     .../node_modules/dtrace-provider install: gyp info spawn args   '/private/var/folders/qc/zygl93h53zb3ctbyw0f0jbzm0000gn/T/ignite-27d4ee3ff9a300fb15dec7abdb4a939b/Foo/node_modules/.pnpm/registry.npmjs.org+dtrace-provider@0.8.8/node_modules/dtrace-provider/build/config.gypi',·
     .../node_modules/dtrace-provider install: gyp info spawn args   '-I',·
     .../node_modules/dtrace-provider install: gyp info spawn args   '/usr/local/lib/node_modules/pnpm/dist/node_modules/node-gyp/addon.gypi',·
     .../node_modules/dtrace-provider install: gyp info spawn args   '-I',·
     .../node_modules/dtrace-provider install: gyp info spawn args   '~/Library/Caches/node-gyp/14.19.0/include/node/common.gypi',·
     .../node_modules/dtrace-provider install: gyp info spawn args   '-Dlibrary=shared_library',·
     .../node_modules/dtrace-provider install: gyp info spawn args   '-Dvisibility=default',·
     .../node_modules/dtrace-provider install: gyp info spawn args   '-Dnode_root_dir=~/Library/Caches/node-gyp/14.19.0',·
     .../node_modules/dtrace-provider install: gyp info spawn args   '-Dnode_gyp_dir=/usr/local/lib/node_modules/pnpm/dist/node_modules/node-gyp',·
     .../node_modules/dtrace-provider install: gyp info spawn args   '-Dnode_lib_file=~/Library/Caches/node-gyp/14.19.0/<(target_arch)/node.lib',·
     .../node_modules/dtrace-provider install: gyp info spawn args   '-Dmodule_root_dir=/private/var/folders/qc/zygl93h53zb3ctbyw0f0jbzm0000gn/T/ignite-27d4ee3ff9a300fb15dec7abdb4a939b/Foo/node_modules/.pnpm/registry.npmjs.org+dtrace-provider@0.8.8/node_modules/dtrace-provider',·
     .../node_modules/dtrace-provider install: gyp info spawn args   '-Dnode_engine=v8',·
     .../node_modules/dtrace-provider install: gyp info spawn args   '--depth=.',·
     .../node_modules/dtrace-provider install: gyp info spawn args   '--no-parallel',·
     .../node_modules/dtrace-provider install: gyp info spawn args   '--generator-output',·
     .../node_modules/dtrace-provider install: gyp info spawn args   'build',·
     .../node_modules/dtrace-provider install: gyp info spawn args   '-Goutput_dir=.'·
     .../node_modules/dtrace-provider install: gyp info spawn args ]·
     .../node_modules/fsevents install: gyp info spawn args   '--generator-output',·
     .../node_modules/fsevents install: gyp info spawn args   'build',·
     .../node_modules/fsevents install: gyp info spawn args   '-Goutput_dir=.'·
     .../node_modules/fsevents install: gyp info spawn args ]·
     .../node_modules/fsevents install: gyp info spawn make·
     .../node_modules/fsevents install: gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]·
     .../node_modules/dtrace-provider install: gyp info spawn make·
     .../node_modules/dtrace-provider install: gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]·
     .../node_modules/dtrace-provider install:   ACTION binding_gyp_ndtp_target_build_ndtp .·
     .../node_modules/fsevents install:   SOLINK_MODULE(target) Release/.node·
     .../node_modules/fsevents install:   CXX(target) Release/obj.target/fse/fsevents.o·
     .../node_modules/fsevents install:   SOLINK_MODULE(target) Release/fse.node·
     .../node_modules/fsevents install: gyp info ok··
     .../node_modules/fsevents install: Done·
     .../node_modules/dtrace-provider install: -----------------------------------------------------------------·
     .../node_modules/dtrace-provider install: Building dtrace-provider failed with exit code 1, falling back
    .../node_modules/dtrace-provider install: on stub implementation.·
     .../node_modules/dtrace-provider install: Re-run install with V set in your environment to see the build·
     .../node_modules/dtrace-provider install: output, or NODE_DTRACE_PROVIDER_REQUIRE=hard to force an·
     .../node_modules/dtrace-provider install: installation failure.·
     .../node_modules/dtrace-provider install: -----------------------------------------------------------------·
     .../node_modules/dtrace-provider install:   TOUCH Release/obj.target/ndtp.stamp·
     .../node_modules/dtrace-provider install: gyp info ok··
     .../node_modules/dtrace-provider install: Done·
     .../node_modules/detox postinstall$ node scripts/postinstall.js·
     .../node_modules/detox postinstall: Detox.framework exists, skipping...·
     .../node_modules/detox postinstall: Done·
     .../node_modules/detox postinstall: Done···
    > foo@0.0.1 postinstall /private/var/folders/qc/zygl93h53zb3ctbyw0f0jbzm0000gn/T/ignite-27d4ee3ff9a300fb15dec7abdb4a939b/Foo
    > node ./bin/postInstall··
     patch-package 6.2.2·
     Applying patches...·
      ELIFECYCLE  Command failed with exit code 1.·
    "

      36 |
      37 |     expect(result).toContain(`Using ignite-cli`)
    > 38 |     expect(result).toContain(`Ignite CLI ignited ${APP_NAME}`)
         |                    ^
      39 |
      40 |     // now let's examine the spun-up app
      41 |     const appPath = filesystem.path(tempDir, APP_NAME)

      at test/vanilla/ignite-new.test.ts:38:20
      at step (test/vanilla/ignite-new.test.ts:33:23)
      at Object.next (test/vanilla/ignite-new.test.ts:14:53)
      at fulfilled (test/vanilla/ignite-new.test.ts:5:58)

Test Suites: 1 failed, 1 passed, 2 total
Tests:       1 failed, 3 passed, 4 total
Snapshots:   0 total
Time:        93.166 s
Ran all test suites matching /.\/test\/vanilla/i.
error Command failed with exit code 1.
```

</details>

jamonholmgren commented:
@tido64 I figured it out ... our CI server installs dependencies in the generated vanilla app via PNPM, to test the symlink support and ensure that we support PNPM (it uses Yarn for Expo, since we haven't totally solved Expo using PNPM yet). If you install PNPM and run `pnpm install` in the `./boilerplate` folder, you'll see a peerDependencies issue.

So -- I don't know as much about this as you likely do. What do you suggest? Yarn and npm install peerDeps automatically, but pnpm fails. Do we do the whole `shamefully-hoist` thing?

tido64 commented:
> So -- I don't know as much about this as you likely do. What do you suggest? Yarn and npm install peerDeps automatically, but pnpm fails. Do we do the whole `shamefully-hoist` thing?

I did this, and it at least fixed the `patch-package` error, and pnpm no longer complains about _missing_ dependencies:

```diff
% git diff package.json
diff --git a/boilerplate/package.json b/boilerplate/package.json
index d0669cca..9616089a 100644
--- a/boilerplate/package.json
+++ b/boilerplate/package.json
@@ -61,6 +61,7 @@
     "@react-native-community/cli-platform-ios": "^6.2.0",
     "@rnx-kit/metro-config": "^1.2.31",
     "@rnx-kit/metro-resolver-symlinks": "^0.1.15",
+    "@storybook/addons": "5.3.21",
     "@storybook/react-native": "5.3.25",
     "@storybook/react-native-server": "5.3.23",
     "@types/i18n-js": "3.0.3",
@@ -85,8 +86,12 @@
     "fbjs-scripts": "3.0.0",
     "jest": "26",
     "jest-circus": "26",
+    "jest-environment-node": "26",
     "jest-expo": "^44.0.1",
     "jetifier": "1.6.6",
+    "metro-config": "0.66.2",
+    "metro-react-native-babel-preset": "0.66.2",
+    "mocha": "6",
     "patch-package": "6.2.2",
     "postinstall-prepare": "1.0.1",
     "prettier": "2.2.1",
@@ -96,9 +101,11 @@
     "react-native-clean-project": "^3.6.3",
     "react-native-web": "^0.16.3",
     "reactotron-core-client": "^2.8.10",
+    "regenerator-runtime": "^0.13.4",
     "solidarity": "2.3.1",
     "ts-jest": "26",
-    "typescript": "4.2.3"
+    "typescript": "4.2.3",
+    "webpack": "4"
   },
   "resolutions": {
     "@types/react": "^17"
```

But I don't know if that's enough to pass CI.

jamonholmgren commented:
I applied that patch, running now.

jamonholmgren commented:
Thanks @tido64 !

jamonholmgren commented:
Hm, it failed on CI, timeout. Running again.

We *really* need to speed up our integration tests.

infinitered-circleci commented:
:tada: This PR is included in version 7.10.10 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/ignite-cli/v/7.10.10)
- [GitHub release](https://github.com/infinitered/ignite/releases/tag/v7.10.10)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
